### PR TITLE
RefreshControl update, Offline Modal, Dumb Animations, Drawer, Vibration

### PR DESCRIPTION
**Pull request by rootical**

### CODE REVIEW

File: App/screens/Login.js [hunk 1]
```diff
@@ -1,87 +1,93 @@
...
-import { Text, TextInput, KeyboardAvoidingView, ScrollView, Button } from 'react-native';
+import {Text, TextInput, KeyboardAvoidingView, ScrollView, Button} from 'react-native';
 import { connect } from 'react-redux';
 import styles from '../theme/ApplicationStyles';
-
-// this is to not remember password, skip it :)
+import OfflineModal from '../components/Offline';
 import {email, password} from '../configs/sensitive';
 import { login } from '../redux/auth';
+import * as Animatable from 'react-native-animatable';
```
Comment (skrywus) [inline App/screens/Login.js:8]: This lib is missing in package.json

File: App/screens/Product.js [hunk 2]
```diff
@@ -1,23 +1,18 @@
...
-    }
-  }
+
   render() {
     const { error, loading, item } = this.props;
     return (
 
       <View style={styles.container}>
-
+        <Button onPress={() => this.props.navigation.goBack()}>Back</Button>
```
Comment (skrywus) [inline App/screens/Product.js]: title in Button is missing. Because of that product detail page is crashing

**Review by skrywus — APPROVED** [review]

File: App/screens/Login.js (hunk 1 above)
Comment (rootical) [reply App/screens/Login.js:8]: It's a dependency of the other, but I will explicitly add it.

File: App/screens/Product.js (hunk 2 above)
Comment (rootical) [reply App/screens/Product.js]: Oh, thanks - it was a blind change :)

**Pull request merged**